Improve React,  Vuejs and Angular integration pages (#3948)
tinymce/tinymce-docs: modules/ROOT/pages/vue-pm-bundle.adoc

= Bundling TinyMCE in Vue.js
:navtitle: Using a package manager with bundling
:description: A guide on bundling TinyMCE with a Vue.js application using a module loader.
:keywords: integration, integrate, vue, vuejs, bundle, webpack, vite
:productSource: package-manager
:packageName: @tinymce/tinymce-vue

> This guide shows how to bundle {productname} into a Vue.js application using the link:https://github.com/tinymce/tinymce-vue[{productname} Vue.js component].

Reviewer: ```suggestion
This guide shows how to bundle {productname} into a Vue.js application using the link:https://github.com/tinymce/tinymce-vue[{productname} Vue.js component^].
```

====
As per the link:https://vuejs.org/about/faq.html#what-s-the-difference-between-vue-2-and-vue-3[Vue FAQ], _Vue 2 will reach End of Life by the end of 2023_.
====

* Follow the prompts and type `+tinymce-vue-demo+` as the project name.

> . Go into the newly created directory.

Reviewer: Navigate is clearer and more formal.
```suggestion
. Navigate into the newly created directory.
```

npm install tinymce "@tinymce/tinymce-vue@^3"
----

. Create `+src/editor.js+` and set the contents to:
+
[source,js]
----
// Add the @tinymce/tinymce-vue wrapper to the bundle
import Editor from '@tinymce/tinymce-vue';

> // Ensure you import TinyMCE to register the global variable required by other components

Reviewer: Remove pronouns `you` 
```suggestion
// Ensure TinyMCE is imported to register the global variable required by other components
```

};
</script>
----

. To start the development server, run:
+
[source,sh]
----
npm run dev
----

== Other resources

* For reporting issues, go to: link:https://github.com/tinymce/tinymce-vue/issues[{productname} Vue.js issues].

> * For more examples of `tinymce-vue` wrapper, see: https://tinymce.github.io/tinymce-vue/[the tinymce-vue storybook].

Reviewer: ```suggestion
* For reporting issues, go to: link:https://github.com/tinymce/tinymce-vue/issues[{productname} Vue.js issues^].
* For more examples of `tinymce-vue` wrapper, see: link:https://tinymce.github.io/tinymce-vue/[the tinymce-vue storybook^].
```
